packages/react/src/components/Slider/Slider.tsx

```diff
@@ -13,7 +13,7 @@ import React, {
 import PropTypes from 'prop-types';
 
 import classNames from 'classnames';
-import throttle from 'lodash.throttle';
+import { throttle } from 'es-toolkit/compat';
```

Comment:
Hey Gui, Thanks for the changes, its Awesome!

I tried to read about throttle from es-toolkit and found that it will accept only 2 parameters like `throttle(func, wait)` while for lodash it's like `throttle(func, wait, options)`

In this file `Slider.tsx` we were using throttle from lodash as
 ```
onDrag = throttle(this._onDrag, EVENT_THROTTLE, {
    leading: true,
    trailing: false,
  });
```

Which means the function `this._onDrag` is executed immediately when the event is triggered , but it will not be executed again at the end of the throttling interval.

Can you please make sure to change the code as well as per es-toolkit to achieve the same functionality.

Reply:
Hey Preeti! 
I tested with lodash and es-toolkit and the behaviour is the same. We are using the `es-toolkit/compat` that has [full compatibility](https://es-toolkit.slash.page/reference/function/throttle.html#compatibility-with-lodash) with lodash, so we don't have to change the code. All `es-toolkit` functions have a "compat" version to ensure we don’t need to change anything in our code.

Reply:
Oh yes, the link is saying it supports `leading` and `trailing` options as well. Thanks!